main/src/com/google/refine/ProjectManager.java

```diff
@@ -307,13 +308,10 @@ public int compare(SaveRecord o1, SaveRecord o2) {
                 "Saving all modified projects ..." :
                 "Saving some modified projects ..."
             );
-
-            for (int i = 0;
-                 i < records.size() &&
-                    (allModified || (LocalDateTime.now().atZone(ZoneId.systemDefault()).toInstant().toEpochMilli() - 
-                            startTimeOfSave.atZone(ZoneId.systemDefault()).toInstant().toEpochMilli() < QUICK_SAVE_MAX_TIME));
+             
+            for (int i = 0;i < records.size() &&
+                    (allModified || (ChronoUnit.MILLIS.between(startTimeOfSave, LocalDateTime.now()) < QUICK_SAVE_MAX_TIME));
                  i++) {
-
```

Comment:
So again this file is full of changes that do not seem related to the feature - if I remember correctly they are meant to address issue #1380 so it looks fine to me, but please use a separate PR next time.